Makefile

```diff
@@ -414,10 +414,12 @@ ifeq ($(KUBECTL_VERSION), $(shell kubectl version --short --client 2>/dev/null |
 	$(info linking local kubectl version $(KUBECTL_VERSION))
 	ln -sf $(shell command -v kubectl) ./local-dev/kubectl
 else
+ifeq ($(KUBECTL_VERSION), $(shell kubectl version --short --client 2>/dev/null | sed -E 's/Client Version: v([0-9.]+).*/\1/'))
```

Comment:
Should this not be neq the local-dev version?

Reply:
:see_no_evil:

Reply:
Turns out there is more than just that wrong with the `local-dev/kubectl` check